docs/examples/button/text.vue

```diff
@@ -1,4 +1,13 @@
 <template>
+  <p>Basic button</p>
+  <div class="flex justify-space-between mb-4">
+    <el-button
+      v-for="button in buttons"
+      :key="button.text"
+      :type="button.type"
+      >{{ button.text }}</el-button
+    >
+  </div>
```

Comment:
Remove this one and create a new file:

> docs/examples/button/link.vue

For demonstrating link type.

Reply:
ok